packages/nodes-base/nodes/Google/YouTube/YouTube.node.ts

```diff
@@ -762,6 +763,14 @@ export class YouTube implements INodeType {
 						qs.type = 'video';
 
 						qs.forMine = true;
+						if (filters.publishedAfter)
+							filters.publishedAfter = DateTime.fromISO(filters.publishedAfter as string)
```

Comment:
Could this break in case the user enters an expression with a date string in another format? or any other type/value
Maybe we could add a check and throw an error if it's not a valid ISO Date string.